src/kit.ts

```diff
@@ -540,14 +603,19 @@ export class KitManager implements vscode.Disposable {
   private _kits = [] as Kit[];
 
   /**
-   * The path to the `cmake-kits.json` file
+   * The path to user-specific the `cmake-kits.json` file
```

Comment:
Just the sentence structure:
`The path to the user-specific `cmake-kits.json` file´